test/CMakeLists.txt

```diff
@@ -64,10 +64,29 @@ target_compile_definitions(ArborX_DetailsAlgorithms.exe PRIVATE BOOST_TEST_MAIN
 target_include_directories(ArborX_DetailsAlgorithms.exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 add_test(NAME ArborX_DetailsAlgorithms_Test COMMAND ./ArborX_DetailsAlgorithms.exe)
 
-add_executable(ArborX_LinearBVH.exe tstLinearBVH.cpp tstKokkosToolsAnnotations.cpp utf_main.cpp)
+set(ARBORX_TEST_QUERY_TREE_SOURCES)
+foreach(_test Degenerated)
```

Comment:
The adjective is `degenerate`, not `degenerated` (which is past form of the verb).